nextflow.config

```
@@ -134,14 +134,43 @@ params {
     bamfiltering_savefilteredbams         = false // can include unmapped reads if --bamfiltering_retainunmappedgenomicbam specified
 
     // Metagenomic Screening
-    run_metagenomicscreening              = false
-    run_metagenomics_complexityfiltering  = false
-    metagenomicscreening_input            = 'unmapped' // mapped, all, unmapped -> mapped vs all specified in SAMTOOLS_FASTQ_MAPPED in modules.conf, unmapped hardcoded SAMTOOLS_FASTQ_UMAPPED
-    metagenomics_complexity_tool          = 'bbduk'
-    metagenomics_complexity_savefastq     = false
-    metagenomics_complexity_entropy       = 0.3
-    metagenomics_prinseq_mode             = 'entropy'
-    metagenomics_prinseq_dustscore        = 0.5
+    run_metagenomics                             = false
+    metagenomics_input                           = 'unmapped' // mapped, all, unmapped -> mapped vs all specified in SAMTOOLS_FASTQ_MAPPED in modules.conf, unmapped hardcoded SAMTOOLS_FASTQ_UNMAPPED
+    run_metagenomics_complexityfiltering         = false
+    metagenomics_complexity_tool                 = 'bbduk'
+    metagenomics_complexity_savefastq            = false
+    metagenomics_complexity_entropy              = 0.3
+    metagenomics_prinseq_mode                    = 'entropy'
+    metagenomics_prinseq_dustscore               = 0.5
+    metagenomics_profiling_tool                  = null
+    metagenomics_profiling_database              = null
+    metagenomics_krakenuniq_ramchunksize         = "16G"
+    metagenomics_kraken_savereads                = false
+    metagenomics_kraken_savereadclassifications  = false
```

```suggestion
    metagenomics_kraken2_savereads                = false
    metagenomics_kraken2_savereadclassifications  = false
```

Make sure to update everywhere